docroot/sites/classrooms.uiowa.edu/modules/classrooms_core/src/Plugin/Block/AdditionalInfoButtons.php

```diff
@@ -28,7 +28,7 @@ public function defaultConfiguration() {
   public function build() {
     $build = [
       '#theme' => 'requestinfobutton_block',
-      '#check_availability_link' => 'http://classroomscheduling.registrar.uiowa.edu/AdAstra7Prod/Portal/GuestPortal.aspx',
+      '#check_availability_link' => 'https://www.aaiscloud.com/uiowa',
```

Comment:
@briand44 can you confirm that this is the old link we are supposed to be replacing?

Reply:
Confirmed